app/components/post_draft/send_button/index.tsx

```diff
@@ -2,14 +2,33 @@
 // See LICENSE.txt for license information.
 
 import {withDatabase, withObservables} from '@nozbe/watermelondb/react';
+import {combineLatest, of as of$} from 'rxjs';
+import {switchMap, distinctUntilChanged} from 'rxjs/operators';
 
 import {SendButton} from '@components/post_draft/send_button/send_button';
 import {Tutorial} from '@constants';
 import {observeTutorialWatched} from '@queries/app/global';
+import {observeLicense} from '@queries/servers/system';
 
-const enhanced = withObservables([], () => {
-    const scheduledPostFeatureTooltipWatched = observeTutorialWatched(Tutorial.SCHEDULED_POST);
+import type {WithDatabaseArgs} from '@typings/database/database';
 
+type Props = WithDatabaseArgs & {
+    scheduledPostEnabled: boolean;
+}
+
+const enhanced = withObservables(['scheduledPostEnabled'], ({scheduledPostEnabled, database}: Props) => {
```

Comment:
Why do we care if enabled or not? This doesn't seem to be doing anything here

Reply:
If you see line number: 25, 

```
if (license?.IsLicensed === 'true' && scheduledPostEnabled) {
```

We are checking if the scheduledPost feature is enabled or not.

Reply:
Aorry missed that one